projects/1.18/assets/create-cafe/createcafe/lang/zh_cn.json

```
@@ -1 +1,170 @@
+  "item.createcafe.starfruit_milk_tea": "杨桃味奶茶",
+  "block.createcafe.starfruit_tea": "杨桃味奶茶",
+  "item.createcafe.blood_orange_milk_tea": "血橙味奶茶",
+  "block.createcafe.blood_tea": "血橙味奶茶",
+  "item.createcafe.lychee_milk_tea": "荔枝味奶茶",
+  "block.createcafe.lychee_tea": "荔枝味奶茶",
+  "item.createcafe.blueberry_milk_tea": "蓝莓味奶茶",
+  "block.createcafe.blueberry_tea": "蓝莓味奶茶",
+  "item.createcafe.pineapple_milk_tea": "菠萝味奶茶",
+  "block.createcafe.pineapple_tea": "菠萝味奶茶",
+  "item.createcafe.apple_milk_tea": "苹果味奶茶",
+  "block.createcafe.apple_tea": "苹果味奶茶",
+  "item.createcafe.pumpkin_milk_tea": "南瓜味奶茶",
+  "block.createcafe.pumpkin_tea": "南瓜味奶茶",
+  "item.createcafe.avocado_milk_tea": "牛油果味奶茶",
+  "block.createcafe.avocado_tea": "牛油果味奶茶",
+  "item.createcafe.sweetberry_milk_tea": "甜浆果味奶茶",
+  "block.createcafe.sweetberry_tea": "甜浆果味奶茶",
+  "item.createcafe.tamarind_milk_tea": "酸角味奶茶",
+  "block.createcafe.tamarind_tea": "酸角味奶茶",
+  "item.createcafe.papaya_milk_tea": "木瓜味奶茶",
+  "block.createcafe.papaya_tea": "木瓜味奶茶",
+  "item.createcafe.mana_berry_milk_tea": "魔力浆果味奶茶",
+  "block.createcafe.mana_tea": "魔力浆果味奶茶",
+  "item.createcafe.kiwi_milk_tea": "猕猴桃味奶茶",
+  "block.createcafe.kiwi_tea": "猕猴桃味奶茶",
+  "item.createcafe.orange_milk_tea": "橙味奶茶",
+  "block.createcafe.orange_tea": "橙味奶茶",
+  "item.createcafe.yucca_milk_tea": "丝兰味奶茶",
+  "block.createcafe.yucca_tea": "丝兰味奶茶",
+  "item.createcafe.aloe_milk_tea": "芦荟味奶茶",
+  "block.createcafe.aloe_tea": "芦荟味奶茶",
+  "item.createcafe.coconut_milk_tea": "椰子味奶茶",
+  "block.createcafe.coconut_tea": "椰子味奶茶",
+  "item.createcafe.persimmon_milk_tea": "柿子味奶茶",
+  "block.createcafe.persimmon_tea": "柿子味奶茶",
+  "item.createcafe.pomegranate_milk_tea": "石榴味奶茶",
+  "block.createcafe.pomegranate_tea": "石榴味奶茶",
+  "item.createcafe.raspberry_milk_tea": "树莓味奶茶",
+  "block.createcafe.raspberry_tea": "树莓味奶茶",
+  "item.createcafe.lemon_milk_tea": "柠檬味奶茶",
+  "block.createcafe.lemon_tea": "柠檬味奶茶",
+  "item.createcafe.watermelon_milk_tea": "西瓜味奶茶",
+  "block.createcafe.watermelon_tea": "西瓜味奶茶",
+  "item.createcafe.vanilla_milk_tea": "奶茶",
+  "block.createcafe.vanilla_tea": "奶茶",
+  "item.createcafe.banana_milk_tea": "香蕉味奶茶",
+  "block.createcafe.banana_tea": "香蕉味奶茶",
+  "item.createcafe.cherry_milk_tea": "樱桃味奶茶",
+  "block.createcafe.cherry_tea": "樱桃味奶茶",
+  "item.createcafe.blackberry_milk_tea": "黑莓味奶茶",
+  "block.createcafe.blackberry_tea": "黑莓味奶茶",
+  "item.createcafe.dragonfruit_milk_tea": "火龙果味奶茶",
+  "block.createcafe.dragonfruit_tea": "火龙果味奶茶",
+  "item.createcafe.gooseberry_milk_tea": "醋栗味奶茶",
+  "block.createcafe.gooseberry_tea": "醋栗味奶茶",
+  "item.createcafe.guava_milk_tea": "番石榴味奶茶",
+  "block.createcafe.guava_tea": "番石榴味奶茶",
+  "item.createcafe.passionfruit_milk_tea": "百香果味奶茶",
+  "block.createcafe.passionfruit_tea": "百香果味奶茶",
+  "item.createcafe.lavender_milk_tea": "薰衣草味奶茶",
+  "block.createcafe.lavender_tea": "薰衣草味奶茶",
+  "item.createcafe.oreo_milk_tea": "奥利奥味奶茶",
+  "block.createcafe.oreo_tea": "奥利奥味奶茶",
```

```suggestion
  "block.createcafe.oreo_tea": "奥利奥味奶茶",
  "item.createcafe.citron_milk_tea": "香橼味奶茶",
  "block.createcafe.citron_tea": "香橼味奶茶",
  "item.createcafe.redlove_milk_tea": "红心果味奶茶",
  "block.createcafe.redlove_tea": "红心果味奶茶",
  "item.createcafe.pomelo_milk_tea": "柚子味奶茶",
  "block.createcafe.pomelo_tea": "柚子味奶茶",
  "item.createcafe.mandarin_milk_tea": "橘子味奶茶",
  "block.createcafe.mandarin_tea": "橘子味奶茶",
  "item.createcafe.barberry_milk_tea": "小檗果味奶茶",
  "block.createcafe.barberry_tea": "小檗果味奶茶"
```
barberry这个口味来自 https://www.curseforge.com/minecraft/mc-mods/powder-more-remastered
推测为物种 _Berberis nummularia_ 的浆果